api-platform/symfony/4.0/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Shouldn't it conflict with api-platform/core < 4.0 ?

Reply:
WDYT @soyuka?